tests/python/relax/test_frontend_from_fx.py

```diff
@@ -19,20 +19,51 @@
 import tvm
 from tvm import relax
 import tvm.testing
-from tvm.script.parser import ir as I, relax as R, tir as T
+from tvm.script import ir as I
+from tvm.script import relax as R
+from tvm.script import tir as T
 
 
-def verify_model(torch_model, input_info, binding, expected):
+def verify_model(
```

Comment:
Question (out scope of this pr) for pt converter verification:  Looks we verify the IR, not very sure if we need to verify the inference result as well?

Reply:
I'm not sure either. I think we don't need it here given we don't have any low-level implementation in the IR after fx translator.